### PR TITLE
Build MTE-756 [v113] Add Taskgraph Integration for Performance Tests

### PR DESCRIPTION
**Pull request by jjSDET**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-756)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/13845)

### Description
We are pulling in the performance tests to Perfherder, and in order to do that need to add new taskcluster tasks as well as update the projects taskcluster dependencies.

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)

### CODE REVIEW

**Comment by isabelrios** [comment]
@jjSDET those are not the rights commits or changes, I firstly added a wrong link in our conversation, the edited it with the correct link to the correct branch... Also, rather than picking the commits, I would suggest you do the changes so that it helps to get familiar with what's needed and maybe why.

File: taskcluster/scripts/bitrise-schedule.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def sync_main(
     parser.add_argument("--commit", required=True, help="the git commit hash to generate screenshots from")
     parser.add_argument("--workflow", required=True, help="the bitrise workflow to schedule")
     parser.add_argument("--artifacts-directory", required=True, help="The directory to store bitrise artifacts")
-    parser.add_argument("--importLocales", dest="locales", metavar="LOCALE", action="append", required=True, help="locale to generate the screenshots for (can be repeated)")
+    parser.add_argument("--importLocales", dest="locales", metavar="LOCALE", action="append", help="locale to generate the screenshots for (can be repeated)")
```
Comment (isabelrios) [inline taskcluster/scripts/bitrise-schedule.py:56]: In addition to this change, in L106 below, you need to add this: 
```
if locales:
        moz_locales_value = " ".join(locales)
 else:
        moz_locales_value = "en-US"
```
or something similar when there are not locales in the arguments. The locales is passed for the screenshots tests.

File: taskcluster/scripts/bitrise-schedule.py [hunk 2]
```diff
@@ -100,10 +100,13 @@ async def async_main(token, branch, commit, workflow, artifacts_directory, local
             await download_log(client, build_slug, artifacts_directory)
 
 
-async def schedule_build(client, branch, commit, workflow, locales, derived_data_path=None):
+async def schedule_build(client, branch, commit, workflow, locales=None, derived_data_path=None):
```
Comment (isabelrios) [inline taskcluster/scripts/bitrise-schedule.py:103]: 👍

File: taskcluster/scripts/bitrise-schedule.py (hunk 1 above)
Comment (isabelrios) [inline taskcluster/scripts/bitrise-schedule.py:56]: 👍

**Comment by isabelrios** [comment]
Hey @JohanLorenzo, this is the scond part of our work to integrate the performance tests results in perferhder via taskcluster and treeherder. ([Document](https://docs.google.com/document/d/13dNoqTcE5V6CIlamcvjx1VEmK4IQeeOIqL6PAg1lvG8/edit#) with all the info for reference)

With this, if we are not wrong, the pending tasks are:

- Schedule the cron. It should be done in mozilla-central
- Connect treeherder to listen to iOS repo. It should be done in treeherder

Could you please help us with those? Not sure if we can do the changes or need someone with access to those repos. Thank you in advance! 🙏

**Comment by isabelrios** [comment]
@jjSDET my commits should be removed from this PR, those add changes that we don't want here... Could you please update this PR with that change? Thanks!

**Comment by JohanLorenzo** [comment]
Nice! 

>     * Schedule the cron. It should be done in mozilla-central

Just to make sure I understand: what's the main reason for scheduling the cron on `mozilla-central`? 

> 
>     * Connect treeherder to listen to iOS repo. It should be done in treeherder

Sure! It's a matter of:
 * Adding the repo just like in https://github.com/mozilla/treeherder/pull/7523
 * Adding this scope to the decision task https://github.com/mozilla-mobile/firefox-android/blob/bb4a6ba0834295163ca4a99aea67b279daaf9199/.taskcluster.yml#L204 
 * Then adding this `extra` block https://github.com/mozilla-mobile/firefox-android/blob/bb4a6ba0834295163ca4a99aea67b279daaf9199/.taskcluster.yml#L321-L336

**Review by JohanLorenzo — APPROVED** [review]
Looks great to me! Great job 👍

**Comment by isabelrios** [comment]
> Nice!
> 
> > * Schedule the cron. It should be done in mozilla-central
> 
> Just to make sure I understand: what's the main reason for scheduling the cron on `mozilla-central`?

oh I was looking at this link in the doc and though that was the place https://hg.mozilla.org/ci/ci-configuration/file/tip/grants.yml#l1750

Thank you for the info for the next steps! 
@jjSDET please let us know if you need more details, help with those.

**Comment by isabelrios** [comment]
@jjSDET I think we can add the two points that Johan's mention in this same PR, modifying the .[taskcluster.yml](url) file:
- Adding this scope to the decision task https://github.com/mozilla-mobile/firefox-android/blob/bb4a6ba0834295163ca4a99aea67b279daaf9199/.taskcluster.yml#L204
- Then adding this extra block https://github.com/mozilla-mobile/firefox-android/blob/bb4a6ba0834295163ca4a99aea67b279daaf9199/.taskcluster.yml#L321-L336

**Comment by isabelrios** [comment]
As @jjSDET pointed out, looks like those lines are already in firefox .taskcluster.yml file:
- Scope: https://github.com/mozilla-mobile/firefox-ios/blob/38de0f774f532065dbedd564be62f32d786ca72e/.taskcluster.yml#L157 
- Extra block: https://github.com/mozilla-mobile/firefox-ios/blob/38de0f774f532065dbedd564be62f32d786ca72e/.taskcluster.yml#L277

We can land this PR then, thanks!

**Comment by adudenamedruby** [comment]
This PR is marked as 113. Is this something that needs to be backported to 113? I don't have context.

**Pull request merged**